### ISSUE TITLE
CSV::MalformedCSVError: Illegal quoting in line 1 with SmarterCSV

### ISSUE THREAD

**Issue by jordansissel** [logstash-plugins/logstash-filter-csv#10]
(This issue was originally filed by @Rajasree at https://github.com/elastic/logstash/issues/3045)

---

I have an issue, when trying to process a csv file, using SmarterCSV. The error I get is -

CSV::MalformedCSVError: Illegal quoting in line 1
This is where the code I use to process the csv file

SmarterCSV.process(file_path)
I have gone through similar questions. But no where I find a good fit that could help me.

I tried to resolve it using some options of SmarterCSV such as -
:remove_empty_values, :remove_empty_hashes etc. But in vain.

I welcome the suggestions or refactoring to make this work? Thanks all

**Comment by harryo1968** [comment]
This issue, in my case, looks like the cause is a poorly crafted header from a custom website.  I just noticed the bad html being generated by a request for a graphic resource:

![logstash_errors1](https://cloud.githubusercontent.com/assets/2893475/9333850/a2d3a60e-4580-11e5-8b67-8626ac9ffa0f.png)

I realized that the issue is triggered by a process outside my scope.  So I have to chase down the developer to tell them to fix their links or it might break my logging tools. :(
